utils/kubernetes/error.go

```diff
@@ -2,6 +2,13 @@ package kubernetes
 
 import "github.com/layer5io/meshkit/errors"
 
+const (
+	ErrNewKubeClientCode = "test_code"
```

Comment:
MESHSYNC1000 maybe?

Reply:
Right now this is not being used anywhere. May be after we decide on the error code strategy? @leecalcote